### PR TITLE
chore: Optimize the situation when code is null

### PR DESCRIPTION
**Pull request by ErKeLost**

### CODE REVIEW

File: src/core/executer.ts [hunk 1]
```diff
@@ -50,7 +50,7 @@ export class Executer {
         process.on('exit', this.closeChild);
 
         child.on('exit', (code) => {
-            this.logger.info(`"${name}" PID ${child.pid} ${code === 0 ? 'done' : `exit ${code}`}`);
```
Comment (shulandmimi) [inline src/core/executer.ts:53]: The code may be null when the execution is successful, so change it to ```${!code ? 'done' : `exit ${code}`}```. Is this better?

**Review by shulandmimi — CHANGES_REQUESTED** [review]
The title in `README.md` needs to be changed from farm to farmup. Can you please change it?

```html
<h1 align="center">
Farm
</h1>
```
Farm => Farmup

File: src/core/executer.ts (hunk 1 above)
Comment (ErKeLost) [reply src/core/executer.ts:53]: do

Comment (ErKeLost) [reply src/core/executer.ts:53]: done

**Pull request merged**